core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -150,6 +150,11 @@ public boolean isSpecial() {
         }
     }
 
+    public static final int MIN_STANDARD_VERSION = 1;
+    public static final int CURRENT_VERSION = MIN_STANDARD_VERSION;
+    public static final int TIMELOCK_VERSION = 2;
+    public static final int MAX_STANDARD_VERSION = 3;
```

Comment:
I think we will need another constant for version 3 that is used for the rules in Risk Analysis, but rules that pertain to version 3 transactions.

in DIP 2, the version 3 transaction is defined and is called "Special"
https://github.com/dashpay/dips/blob/master/dip-0002.md

So maybe we should have a `DIP2_VERSION = 3`, in a similar way that `TIMELOCK_VERSION = 2`

Reply:
`DIP2_VERSION` could instead be called `SPECIAL_VERSION` -- which seems like a strange name, but `Special` is used in some class names that relate to version 3 transactions.

Reply:
Makes sense. I like "special" more :)